crates/rattler_networking/src/lib.rs

```diff
@@ -20,12 +22,24 @@ pub struct AuthenticatedClient {
     auth_storage: AuthenticationStorage,
 }
 
-impl Default for AuthenticatedClient {
+/// Returns the default auth storage directory used by rattler.
+/// Would be placed in $HOME/.rattler, except when there is no home then it will be put in '/rattler/'
+pub fn default_auth_store_fallback_directory() -> &'static Path {
+    static FALLBACK_AUTH_DIR: OnceLock<PathBuf> = OnceLock::new();
+    FALLBACK_AUTH_DIR.get_or_init(|| {
+        dirs::home_dir()
+            .map(|home| home.join(".rattler/"))
+            .unwrap_or_else(|| {
+                tracing::warn!("Home directory could not be found, using '/rattler'");
```

Comment:
```suggestion
                tracing::warn!("using '/rattler' to store fallback authentication credentials because the home directory could not be found");
```